bin/lambda_handler.rb

```diff
@@ -6,7 +6,7 @@
 def lambda_handler(event:, context:)
   report_name = event['report']
 
-  return { statusCode: 400, body: "No report was specified" } if report_name.nil?
+  raise 'No report was specified' if report_name.nil?
```

Comment:
Totok. Lambda by mala crashnut, ak nema report_name.
Detto ak bude chyba v parseri/fetch.

Reply:
@samuelszabo preco?

Reply:
Lebo lambda má failnúť, ak má ostať entry v queue. Lambda ako taká, ak vráti 400, tak je to považované za OK stav (entry z queue sa zmaže).

Reply:
![nasta-backend - Lambda](https://user-images.githubusercontent.com/403439/76314720-a1c8bf00-62d7-11ea-80a9-84369876cb24.png)

O tomto v podstate hovorím. 404 ale Success.